src/echo/subscriber.js

```diff
@@ -35,6 +35,18 @@ import {
 
 import * as CacheMaps from '../constants/cache-maps';
 
+/**
+ * @param {Error} error
+ * @param {() => any} [handler]
+ */
+function handleConnectionClosedError(error, handler) {
```

Comment:
maybe add helper in utils ?

Reply:
This file is only place, where this handler is needed.